source/agora/network/Manager.d

```diff
@@ -981,7 +981,7 @@ public class NetworkManager
                     headers.map!(header =>
                         tuple(header.height,
                             iota(0, enrolled_validators[header.height]).filter!(i =>
-                                header.validators[i]).count()
+                                header.validators[i] || header.missing_validators.find(i)).count() // Take into account the slashed validators
```

Comment:
Shouldn't this be `canFind`? AFAIK `find` returns a range, and I'm not sure how well empty ranges convert to `bool`.

Reply:
My mistake, I was thinking it was bool for `find`. Will update to `canFind`.

Reply:
Updated